imagersite/imager_images/templates/imager_images/library.html

```diff
@@ -36,27 +51,42 @@ <h2 class="text-center text-capitalize">{{ user }}'s Library</h2>
                 </section>
             </div>
             <br>
-            <div class="row library">
-                <section class="col-md-10 col-md-offset-1">
+            <div class="row">
+                <section class="col-md-10 col-md-offset-2">
                     <h3> Albums </h3>
+                        <div class="row library"
                         <ul class="list-unstyled">
                             {% for album in albums %}
-                                <li>
+                                <li class="col-md-5 list-unstyled" style="padding: 5%;">
                                     <a href="{% url 'album_detail' album.id %}">
                                         <img src="{{ album.cover.url }}" width="300px"/>
                                     </a>
                                 </li>
                             {% endfor %}
                         </ul>
+                        </div>
+                </section>
+            </div>
+            <div class="row text-center">
+                <section class="pagination text-center">
+                    <span class="step-links">
+                        {% if albums.has_previous %}
+                        <li class="list-unstyled"><a class="btn btn-default" href="?page={{ albums.previous_page_number }}">Previous</a></li>
+                        {% endif %}
+                        {% if albums.has_next %}
+                        <li class="list-unstyled"><a class="btn btn-default" href="?page={{ albums.next_page_number }}">Next</a></li>
```

Comment:
You only have one parameter for pagination applying to both albums and photos. What if I wanted to go forward through the list of photos, but I don't want to change the page of albums?